dispatcher.py

```diff
@@ -116,6 +117,23 @@ def CreateJobSummaryTemplateValues(rebuilt_images, summary):
 
     github_token = os.getenv("GITHUB_TOKEN")
 
+    helm_repo_creds = {}
+    helm_repo_creds["artifactory.algol60.net"] = {
```

Comment:
I decided to do it this way just in case if in the future we have multiple private helm repos in the future.